### ISSUE TITLE
Auto fill or dropdown server url input field on logout

### ISSUE THREAD

**Issue by BaccanoMob** [jmshrv/finamp#876]
Currently when you logout of the app, you will get the initial startup screen (as if a fresh install excluding if there are downloads). 

Because of #870 I had to logout and login multiple times and had to retype in the url several times. I think if the field should use the last used server url or have list of server url in a drop down with x on side to delete if needed. Personal opinion is to use the last used url since I doubt people use multiple instance of jellyfin at a time.

This feature will be useful for desktop (or TV if planned/and typing in TV is a pain) apps where multiple users use the same app.

**Comment by Chaphasilor** [comment]
I agree that the login flow could be improved. Instead of dropping back to first screen, we could just go back to the user selection / authentication screen instead, keeping the server settings the same unless the user manually goes back?

**Comment by BaccanoMob** [comment]
That's even better.

But I would still prefer it pre filled (I think the variable of name BASEURL, should be set as initial value) from setting just in case if someone goes back accidentally. Prolly with x symbol in the right end to clear if they do need to change it. Well, if exiting and reopening the app (when logged out) would still go to the auth screen, you can ignore this.
